src/index.js

```diff
@@ -58,11 +58,13 @@ export default class Client {
     return url;
   }
 
-  _urlencode = function(obj) {
+  _urlencode(obj) {
```

Comment:
function and methods should be `camelCase`

Reply:
I'd updated this last week, I will push the new changes

Reply:
this a private method though, is there a need for camelCase